reg_tests/CTestList.cmake

```diff
@@ -121,63 +121,22 @@ if(NOT ENABLE_CUDA)
   add_test_r(ablNeutralEdgeSegregated 8)
   add_test_r(ablStableEdge 4)
   add_test_r(ablNeutralEdgeNoSlip 4)
-  add_test_r(ablStableElem 4)
   add_test_r(ablUnstableEdge 4)
   add_test_r(ablUnstableEdge_ra 4)
   add_test_r(airfoilRANSEdgeNGPTrilinos.rst 1)
   add_test_r(conduction_p4 4)
-  add_test_r(cvfemHC 8)
-  add_test_r(dgMMS 6)
-  add_test_r(dgNonConformal 4)
-  add_test_r(dgNonConformal3dFluids 4)
-  add_test_r(dgNonConformal3dFluidsHexTet 4)
-  add_test_r(dgNonConformal3dFluidsP1P2 8)
-  add_test_r(dgNonConformalEdge 4)
-  add_test_r(dgNonConformalEdgeCylinder 8)
-  add_test_r(dgNonConformalElemCylinder 8)
-  add_test_r(dgNonConformalFluids 4)
-  add_test_r(dgNonConformalFluidsEdge 4)
-  add_test_r_rst(dgNonConformalThreeBlade 4)
   add_test_r(drivenCavity_p1 4)
-  add_test_r(ductElemWedge 2)
-  add_test_r(ductWedge 2)
   add_test_r(edgeHybridFluids 8)
   add_test_r(ekmanSpiral 4)
-  add_test_r(ekmanSpiralConsolidated 4)
-  add_test_r_inp(elemBackStepLRSST 4)
-  add_test_r(elemClosedDomain 2)
-  add_test_r(elemHybridFluids 8)
-  add_test_r(elemHybridFluidsShift 8)
-  add_test_r(femHC 2)
-  add_test_r(femHCGL 2)
-  add_test_r(heatedBackStep 4)
```

Comment:
I'm wondering if we should convert a few more of these tests to be edge scheme. Browsing through, I'm thinking about:

- `closedDomain`
- at least one of the `backStep`'s
- `milestoneRun`
- `variableDensNonIso` (this might be better to swap the with the NonUniform version of the test which uses mixture fraction. IIRC we're going to remove mixture fraction at some point as well)

Reply:
When we reinstate these tests, we should rename tests like `milestoneRun` with something more relatable

Reply:
Agreed